.github/workflows/release.yaml

```diff
@@ -0,0 +1,71 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
+          echo ::set-output name=version::${VERSION}
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Create scalardb test containers
+        run: ./gradlew docker -PdockerVersion=${{ steps.version.outputs.version }}
+        working-directory: scalardb-test
+
+      - name: Create scalardl test containers
+        run: ./gradlew docker -PdockerVersion=${{ steps.version.outputs.version }}
+        working-directory: scalardl-test
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - name: Push containers
+        run: |
+          docker push ghcr.io/scalar-labs/kelpie-test-scalardb:${{ steps.version.outputs.version }}
+          docker push ghcr.io/scalar-labs/kelpie-test-scalardl:${{ steps.version.outputs.version }}
+
+      - name: Create release
```

Comment:
Do we need to create a release? Maybe only pushing the docker image is enough?

Reply:
@brfrn169 
You're right. I have corrected it.
https://github.com/scalar-labs/kelpie-test/pull/42/commits/f7d3f496d8e496d274210b6e44b0ade83f4d97ea